src/pages/Introduction/Introduction.scss

```diff
@@ -1,5 +1,6 @@
 .introduction {
     height: 100vh;
```

Comment:
Consider using a more descriptive class name for the introduction section to improve code readability. 

Suggested change:
```scss
.introduction-section {
``` 

For reference, see [CSS Naming Conventions](https://en.bem.info/methodology/naming-convention/).